src/components/panel/panel.js

```diff
@@ -1137,6 +1152,53 @@ MdPanelRef.prototype._configurePosition = function() {
 
 
 /**
+ * Focuses on the panel or the first focus target.
+ * @private
+ */
+MdPanelRef.prototype._focusOnOpen = function() {
+  if (this._config['focusOnOpen']) {
+    // Wait a digest to guarantee md-autofocus has finished adding the class
+    // _md-autofocus, otherwise the focusable element isn't available to focus.
+    var self = this;
+    // TODO(KarenParker): Use $$postDigest here.
+    this._$rootScope.$applyAsync(function() {
+      var target = self._$mdUtil.findFocusTarget(self._panelEl) ||
+        self._panelEl;
+      target.focus();
+    });
+  }
+};
+
+
+/**
+ * Shows the backdrop.
+ * @returns {!angular.$q.Promise} A promise that is resolved when the backdrop
+ * is created and attached.
+ * @private
+ */
+MdPanelRef.prototype._createBackdrop = function() {
+  if (this._config.hasBackdrop) {
+    var backdropAnimation = this._$mdPanel.newPanelAnimation()
+      .openFrom(this._config.attachTo)
+      .withAnimation({
+        open: '_md-opaque-enter',
+        close: '_md-opaque-leave'
+      });
+    var backdropConfig = {
+      animation: backdropAnimation,
+      attachTo: this._config.attachTo,
+      focusOnOpen: false,
+      panelClass: 'md-panel-backdrop',
```

Comment:
I think this should be a private class. We don't want people modifying the backdrop.